app/parsers/powershell.py

```diff
@@ -0,0 +1,13 @@
+import logging
+from app.utility.base_parser import BaseParser, PARSER_SIGNALS_FAILURE
+
+
+class Parser(BaseParser):
+
+    def parse(self, blob):
+        for ex_line in self.line(blob):
+            if 'FullyQualifiedErrorId' in ex_line:
+                log = logging.getLogger('parsing_svc')
+                log.warning('This ability failed for some reason. Manually updating the link to report a failed state.')
```

Comment:
Should this be `An ability failed...` or will the log message reference which ability it was?

Reply:
This parser is specifically for all atomic style powershell abilities. I can see a case where we would want to use this elsewhere, but that's something to think about later. As for referencing which ability encountered the error, the other half of the changes print out those details (that way you don't have to worry about handling any of that here).